pkgs/development/python-modules/pydrive2/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib,
+  python3,
+  fetchPypi,
+  google-api-python-client,
+  oauth2client,
+  pyopenssl,
+  pyyaml,
+  six,
+ }:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "pydrive2";
+  version = "1.10.0";
+
+  src = fetchPypi {
+    pname = "PyDrive2";
+    inherit version;
+    sha256 = "1av3fkbq8xvrvshy8yiq6ysllwll1mfcaspv4aww1xhyzys1kggp";
+  };
+
+  # the tests require GCP credentials
+  doCheck = false;
```

Comment:
Please add a pythonImportsCheck instead and move doCheck after all *buildInputs.

Reply:
fixed in decb45313390f19369ceeea3d34700add2c10700